spa/src/components/Main.js

```diff
@@ -12,15 +12,16 @@ import { useConfigureAnalytics } from './analytics';
 import { CONTENT_SLUG, VERIFY_EMAIL_SUCCESS } from 'routes';
 import PageContextProvider from './dashboard/PageContext';
 
+import { useUserContext } from './UserContext';
+
 const FeatureFlagsProviderContext = createContext(null);
-const UserDataProviderContext = createContext(null);
 
 function Main() {
   useConfigureAnalytics();
-
   const [loadingFlags, setLoadingFlags] = useState(true);
   const [featureFlags, setFeatureFlags] = useState();
-  const [userData, setUserData] = useState();
+  const [user, setUser] = useState();
+  const { setUser: setUserContext } = useUserContext();
```

Comment:
Can't you use `user` from Context as well?
```suggestion
  const { user, setUser } = useUserContext();
```

Reply:
done